### PR TITLE
chore: mmr args

### LINKED ISSUE #110: Darwinia > Ethereum 领取RING时手续费过高
### Prerequisites

- [ ] Can you reproduce the problem in safe mode?
- [X] Are you running the latest version?

### Importance

Serious

### Priority

High

### Description

Darwinia > Ethereum 领取RING时手续费$4500
领取时先弹出合约地址错误的提示，然后进入metamask后手续费过高。
image
image

### Please provide the environment you discovered this bug in

_No response_

### Anything else?

修复过后，需 @xiaoch05 查看，若没有问题可以立即推入生产环境。

### PR DESCRIPTION
**Pull request by sxlwar**
## Priority

Do you need reviewers to review this PR as soon as possible?

- [ ] High
- [x] Medium
- [ ] Low

## Help Merge

Do you need any other people merge it if all check passed and review approved?

- [x] Yes - Merge it immediately
- [ ] NO - Please let me merge

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Why do we need this pr?
maybe fix #110

### CODE REVIEW

**Review by WoeOm — APPROVED** [review]

File: src/utils/records/d2e.ts [hunk 1]
```diff
@@ -99,63 +100,51 @@ export function claimToken({
...
-          blockHeader: header.toHex(),
-          peaks: mmrProof.peaks,
-          siblings: mmrProof.siblings,
-          eventsProofStr: eventsProof.toHex(),
-        },
-      })).pipe(
-        switchMap(({ account, proof }) => {
-          const { root, MMRIndex, blockHeader, peaks, siblings, eventsProofStr } = proof;
-          const contractAddress = toNetworkConfig.tokenContract.bankEthereum || '';
+      return MMRRoot && best && best > blockNumber
```
Comment (HackFisher) [inline src/utils/records/d2e.ts:105]: Need more documents here.

File: src/utils/records/d2e.ts [hunk 2]
```diff
@@ -99,63 +100,51 @@ export function claimToken({
 
   return apiObs.pipe(
     switchMap((api) => {
-      const mmrProofObs = from(getMMRProof(api, blockNumber, mmrIndex, blockHash));
-      const eventsProofObs = from(getMPTProof(api, blockHash, storageKey));
+      const eventsProofObs = from(getMPTProof(api, blockHash, storageKey)).pipe(map((str) => str.toHex()));
```
Comment (HackFisher) [inline src/utils/records/d2e.ts:103]: @WoeOm I remember there was a storage key prefix change(runtime upgrade like a hardfork to chain relay), did we  need to check that here?

Comment (WoeOm) [reply src/utils/records/d2e.ts:103]: https://github.com/darwinia-network/wormhole-ui/blob/19635be932dfe4a4244bd6e78a70d4b25784d036/src/utils/records/d2e.ts#L149

在这边处理的

Comment (sxlwar) [reply src/utils/records/d2e.ts:103]: const storageKey = getD2ELockEventsStorageKey(blockNumber, config.lockEvents);

Line 91

As this， the storage key has already been corrected

**Pull request merged**